### ISSUE TITLE
Native JS errors don't properly serialize

### ISSUE THREAD

**Issue by Exponential-Workload** [open-runtimes/open-runtimes#204]
Currently, JSON.stringify() is being called on the object passed to ctx.error() if it's an Object - which sadly does not work on Error classes, as they serialize to `{}`.
This results in `ctx.error(new Error('some message'))` outputting `{}`, which is sure to confuse people, and I'm guessing isn't intended behaviour.

A minimal patch would be to check if an object is an instanceof Error, and if so, return `err.stack ?? err.message`.

**Comment by renzo-s** [comment]
Is there an update for this issue / PR? Seems like a simple feature to implement without breaking anything.

**Comment by Exponential-Workload** [comment]
> Is there an update for this issue / PR? Seems like a simple feature to implement without breaking anything.

Nope, last I heard was https://github.com/open-runtimes/open-runtimes/pull/205#issuecomment-1973569588
